styles.css

```diff
@@ -0,0 +1,184 @@
+body{
+  font-family: Open Sans, sans-serif;
+  background-color: #EAEAF4;
+}
+
+.sidebar{
```

Comment:
This is all looking good! I say let's make sure to check the style guide to see if spaces are needed in spots like this one.